tern/analyze/docker/dockerfile.py

```diff
@@ -254,6 +254,13 @@ def find_git_info(line, dockerfile_path):
     logger.debug('Parsed src_path is %s', src_path)
     # get the git project info
     comment_line = common.check_git_src(dockerfile_path)
+    # get the git project link
+    url_list = common.get_git_url(dockerfile_path)
+    if url_list:
+        comment_url = ''
+        for url in url_list:
+            comment_url += url + ' '
```

Comment:
@nishakm  small nit -- do we want to separate the URLs with nothing but a space? I would think if there is a list of URLs we would want to separate by comma, but maybe that is getting too detailed :)

Reply:
@rnjudge Good catch! I didn't notice. @ForgetMe17 can you make this change?

Reply:
Yes!

Reply:
The output now is (2 lines)
`FROM ubuntu@sha256:04d48df82c938587820d7b6006f5071dbbffceb7ca01d2814f81857c631d44df`
`ADD plain_file /tmp # git project name: tern, HEAD sha: 7009ee03938a4f18b9e884f101a4d1bb999a36f3, project url(s): github.com/vmware/tern, github.com/ForgetMe17/tern`